.github/actions/add-package/action.yml

```diff
@@ -66,7 +83,7 @@ runs:
 
         for cnt,asset in enumerate(assets):
           delete = False
-          if cnt >= 30:
```

Comment:
I think the number was incorrect. Before we wanted to keep around the latest 8 builds, and now the latest 4 builds (since now CI generates 2 versions per platform). In both cases, we need to keep the assets while `cnt < 32` (`enumerate` starts counting from `0`), and delete them when `cnt >= 32`.

Reply:
Makes sense, good catch!